### PR TITLE
Actually enables monkey

### PR DESCRIPTION
**Pull request by ghost**
OOGA OOGA AHHH AHHH

### CODE REVIEW

**Comment by 81Denton** [comment]
>+#IC_PRINTING
🤔

**Comment by MoreRobustThanYou** [comment]
@81Denton that's just the regular github newline fuckery, ignore it, nothing changes on that line

**Comment by coiax** [comment]
This won't enable monkey on the server, only people with remote can change that.

**Comment by MoreRobustThanYou** [comment]
yes ik

**Comment by MoreRobustThanYou** [comment]
but this is just to get opinions on it, ya see?

**Comment by praisenarsie** [comment]
it got closed last time mrty....

File: config/game_options.txt [hunk 1]
```diff
@@ -492,4 +492,4 @@ MICE_ROUNDSTART 10
 #EMERGENCY_SHUTTLE_AUTOCALL_THRESHOLD 0.2
 
 ## Determines if players are allowed to print integrated circuits, uncomment to allow.
-#IC_PRINTING
\ No newline at end of file
+#IC_PRINTING
```
Comment (Armhulen) [inline config/game_options.txt]: undocumented change

**Review by ghost — COMMENTED** [review]
stop it IDIOT

File: config/game_options.txt (hunk 1 above)
Comment (MoreRobustThanYou) [reply config/game_options.txt]: it hasn't changed. the last line is always green-red in github changes.

```
-#IC_PRINTING
+#IC_PRINTING
```

literally nothing changes

**Comment by KorPhaeron** [comment]
lmai

**Comment by ShiggyDiggyDo** [comment]
enable meteor

**Comment by Buggy123** [comment]
fucking finally :+1:

and disable revs while you're at it. and re-enable gang. and add shadowlings again, and we'll finally have some decent gamemodes.

**Comment by MoreRobustThanYou** [comment]
@Buggy123 gangs got scrubbed completely from the code because it was shit

**Comment by lzimann** [comment]
1) This won't do anything on any server.
2) Monkey mode is still fundamentally a broken gamemode and is honestly a dead game mode(unless someone actually overhauls it), leaving it on by default is not right.

**Comment by Buggy123** [comment]
@MoreRobustThanYou so did sling

File: config/game_options.txt (hunk 1 above)
Comment (JJRcop) [reply config/game_options.txt]: line endings etc

**Comment by MoreRobustThanYou** [comment]
@Buggy123 sling is good

**Pull request closed without merging**